### PR TITLE
Use interstitial screen for the education screen

### PR DESCRIPTION
**Pull request by chung-lai**
As the new interstitial screen with instructions combined perform better and have better UX. We are changing the existing education screens with the new interstitial.

### How was this tested?
- [ ] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### Manual test cases
If manually tested, describe the different scenarios you followed to ensure
that your changes work.  Make sure that you've tested as many pathways as
possible -- not just the [happy path](https://en.wikipedia.org/wiki/Happy_path).

### Screenshots and screen recordings

* Settings *

Screen Shot 2022-03-21 at 1 45 41 PM

* Promo Card *

Screen Shot 2022-03-21 at 1 45 50 PM

## Issues addressed
* Link(s) to Github issue(s) (if applicable)

### CODE REVIEW

**Review by hwiatneeva — COMMENTED** [review]
LGTM from design. SWE should approve the code changes. 

Thanks much, Chung!!!!

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 1]
```diff
@@ -1693,7 +1693,9 @@ extension BrowserViewController {
                 } buttonAction: {
                     self.overlayManager.hideCurrentOverlay()
                     UIApplication.shared.openSettings(
-                        triggerFrom: .defaultBrowserPromptMergeEduction
+                        triggerFrom: fromSkipToBrowser
+                            ? .defaultBrowserPromptSkipToBrowser
+                            : .defaultBrowserPromptMergeEduction
```
Comment (darinwf) [inline Client/Frontend/Browser/BrowserViewController.swift:1698]: I can see it getting a bit complicated when we want to convey state like this through a series of actions. Probably keeping it simple as you have done is the way to go for now. Makes me wonder if there is a better way though.

File: Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserIntestitialOnboardingView.swift [hunk 2]
```diff
@@ -6,8 +6,74 @@ import Defaults
 import Shared
 import SwiftUI
 
+class DefaultBrowserInterstitialOnboardingViewController: UIHostingController<
```
Comment (darinwf) [inline Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserIntestitialOnboardingView.swift:9]: We should probably be using the OverlayManager these days. Probably a question for Evan though.

**Review by darinwf — APPROVED** [review]

File: Client/Frontend/Browser/BrowserViewController.swift (hunk 1 above)
Comment (chung-lai) [reply Client/Frontend/Browser/BrowserViewController.swift:1698]: Yes, agree. Would love to spend a bit of time to think about the state as the default browser interstitial stablize.

File: Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserIntestitialOnboardingView.swift (hunk 2 above)
Comment (chung-lai) [reply Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserIntestitialOnboardingView.swift:9]: Yes, I tried a few options with OverlayManager and couldn't get one with full sheet presented. Let me add a TODO to figure this out with Evan about whether we have support on that.

**Pull request merged**